LayoutTests/fast/images/spatial-image-controls-expected.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
We need to test how this feature interacts with ImageOverlay.
See tests in LayoutTests/fast/images/text-recognition.
We need to add a test scenario in which both this feature and text recognition feature kicks in.

Reply:
The tests in LayoutTests/fast/images/text-recognition all fail on visionOS when I tried running them (except the ones about text recognition NOT running). I assume text recognition isn't supported on visionOS.